docs/modules/ROOT/partials/k3s_kubeconfig_keycloak.adoc

```diff
@@ -1,16 +1,33 @@
-== Get kubeconfig and admin password
+== Get kubeconfig and Keycloak users credentials
 
 Retrieve the Kubeconfig file:
 
 ```shell
-$ terraform output -json kubeconfig | jq -r .
+$ terraform output -json kubeconfig | jq -r . > kubeconfig.yaml
+$ export KUBECONFIG=kubeconfig.yaml
 ```
 
+By default, two users are defined in Keycloak:
 
-Retrieve the Keycloak password for the `jdoe` user of the `devops-stack` realm:
+[cols="a,a,a,a,a",options="header]
+|===
+| user | Keycloak role | Keycloak realm | terraform output | comment
 
-```shell
-$ terraform output admin_password
-```
+| admin
+| Administrator
+| all
+| `keycloak_admin_password`
+| This user has admin rights only in Keycloak. Use `keycloak_url` and select "Administration Console" to login.
 
-You will use this user and password to log in to applications.
+| jdoe
+| applications
+| Kubernetes
+| `jdoe_password`
+| This user has related applications rights within Kubernetes realm. Use `keycloak_url/auth/realms/kubernetes/account/` to login.
```

Comment:
@vquiby the realm was renamed **devops-stack** instead of **kubernetes**.
I also replaced the static jdoe user with a map of users that is passed as a root module argument.
Plz check the following [link](https://github.com/camptocamp/devops-stack/pull/788/files#diff-a2703b9aa89de851a3ef6f0a9aec302576810d851b772ec79475767f8823e73fR13-R23).
For the k3s variant, there is a default map with a single key (jdoe user).